src/install/config/cassandra.yaml

```diff
@@ -0,0 +1,81 @@
+agentName: cassandra
+agentType: integration
+title: 'Cassandra monitoring integration'
+metaDescription: 'Install the Cassandra integration'
+introFilePath: 'src/install/cassandra/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
+      - value: 'linux'
+        displayName: 'Linux'
+        logo: 'linux'
+      - value: 'ecs'
+        displayName: 'ECS'
+        logo: 'amazonECS'
+      - value: 'kubernetes'
+        displayName: 'Kubernetes'
+        logo: 'k8'
+  - optionType: deployment
```

Comment:
I believe this is still missing the steps for Zypper and Tarball

https://docs.newrelic.com/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations/#zypper

https://docs.newrelic.com/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations/#tarball

Reply:
Zypper and Tarball should go in all the integrations?